README.md

```diff
@@ -1,73 +1,92 @@
-# Role Name
+# crypto_policies
 
-A template for an ansible role which configures some GNU/Linux subsystem or
-service. A brief description of the role goes here.
+This Ansible role manages system wide crypto policies.
+
+This concept is well adopted since Red Hat Enterprise Linux 8 and in Fedora.
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should
-be mentioned here. For instance, if the role uses the EC2 module, it may be a
-good idea to mention in this section that the `boto` package is required.
+The system wide crypto policies are implemented and tested on RHEL 8/CentOS 8
+and Fedora.
+
+The role might work elsewhere, but might need enabling in [vars/](vars/).
 
 ## Role Variables
 
-A description of all input variables (i.e. variables that are defined in
-`defaults/main.yml`) for the role should go here as these form an API of the
-role.
+Unconfigured, this role will just report system status as described in the
+following section.
 
-Variables that are not intended as input, like variables defined in
-`vars/main.yml`, variables that are read from other roles and/or the global
-scope (ie. hostvars, group vars, etc.) can be also mentioned here but keep in
-mind that as these are probably not part of the role API they may change during
-the lifetime.
+ * `crypto_policies_policy`
 
-Example of setting the variables:
+Use this variable to specify the target crypto policy on the system, which
+can be either the base policy level or a base policy with policy submodule
+as accepted by the `update-crypto-policies` tool, if both are available
+on the target system, for example `FUTURE` or `DEFAULT:NO-SHA1:GOST`.
+Default is `false` meaning the configuration is not changed and role
+will just collect the facts.
 
-```yaml
-template_foo: "oof"
-template_bar: "baz"
-```
+ * `crypto_policies_reload`
+
+By default (`true`), crypto policies reload most common daemons using crypto
+policies in the system. Setting `false` prevents this behavior and is helpful if
+the role is executed during system enrollment or some other follow-up
+configuration is expected to come later.
+
+ * `crypto_policies_reboot`
+
+Crypto policies can not know all the custom applications using crypto
+libraries that are affected by change of crypto policies. By default
+(`true`), the machine is rebooted after applying changes.
```

Comment:
I'd recommend defaulting this to false. Especially when you have this indicator crypto_policies_reboot_required.

Reply:
I feel like we should be safe to have it on by default (it is actually rebooted only if the policy is changed).

Lets say for one-shot playbooks it will be easier for users to just set one variable and all is done without more bothering with variables.

For more complicated setups with more roles and tasks where one would like to reboot later, I think it is fine requiring the user to set one more variable `crypto_policies_reboot: false` and handle the output variable to reboot later.

Reply:
I changed the default to `false` after discussion with @pcahyna.